classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.classroom = self
```

Comment:
Great job on implementing the `has-many` / `belongs_to` relationship between student and classroom :+1:

Reply:
Thanks! @christianbiring1